tests/Provider/Doctrine/Persistence/Schema/SchemaManagerTest.php

```diff
@@ -291,7 +291,7 @@ public function testComputeAuditTableName(string $tableName, string $expectedRes
     /**
      * @return iterable<string, array<bool|string>>
      */
-    public static function provideTableNames(): iterable
+    public static function provideComputeAuditTableNameCases(): iterable
```

Comment:
Don't know why, but cs-fixer want to change these names
https://github.com/DamienHarper/auditor/actions/runs/17256176602/job/48968354257?pr=247